quiz.css

```diff
@@ -0,0 +1,74 @@
+/*   Initial CSS    */
+
+header.dbg {
+	border-width: 1px;
+	border-style: dotted;
+	border-color: lightgray;
+}
+
+h1 {
+	text-align: center;
+}
+
+img.myImg {
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+p {
+	text-align: justify;
+	text-justify: inter-word;
+	padding-left: 5px;
+	padding-right: 5px;
+}
+
+h4.myH {
+	font-weight: bold;
+	padding-left: 5px;
+	padding-right: 5px;
+}
+
+h4.dgb{
```

Comment:
All your other selectors have a space between the selector and its pair of curly braces